src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -157,9 +167,14 @@ class Diagram extends Component<Props> {
 
   componentDidMount() {
     this.props.fetchFlows()
+    this.setState({ currentLang: this.props.contentLang })
```

Comment:
You could set currentLang in the state declaration instead of initializing it as empty

Reply:
I tried and it didn't work as it's not initiated yet, the only way I could see it working would be in the constructor, but it would end up being similar